crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -633,3 +642,17 @@ async fn get_sequencer_chain_id(client: SequencerClient) -> eyre::Result<tenderm
 fn max_permitted_celestia_height(reference: u64, variance: u64) -> u64 {
     reference.saturating_add(variance.saturating_mul(6))
 }
+
+#[instrument(skip_all)]
+fn report_exit(exit_reason: eyre::Result<&str>, message: &str) -> eyre::Result<()> {
+    match exit_reason {
+        Ok(reason) => {
+            info!(reason, message);
```

Comment:
`tracing` by default uses `Debug` formatting for additional fields, so best to use `%` sigil to get `Display` formatting:
```suggestion
            info!(%reason, message);
```